tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -86,3 +86,34 @@ Wait Until All Builds Are Complete
     FOR    ${build_data}    IN    @{builds_data}
         Wait Until Build Status Is    namespace=${namespace}    build_name=${build_data['metadata']['name']}  expected_status=Complete
     END
+
+Delete Multiple Builds
+    [Documentation]     Deletes Multiple Builds in the given namespace
+    [Arguments]   @{build_name_list}  ${namespace}
+    FOR    ${build}    IN    @{build_name_list}
+        ${build_name}=  Search Last Build  namespace=${namespace}    build_name_includes=${build}
+        Delete Build    namespace=${namespace}    build_name=${build_name}
+    END
+
+Start Remaining Builds And Wait Until All Builds Are Complete
+    [Documentation]    Starts new build if build fails or is not started , Waits until all builds are complete
+    [Arguments]    ${builds}  ${build_configs}
+    ${no_of_builds} =    Get Length    ${builds}
+    FOR    ${ind}    IN RANGE    ${no_of_builds}
+        ${build_name} =    Search Last Build    namespace=redhat-ods-applications
+        ...    build_name_includes=${builds}[${ind}]
+        IF    "${build_name}" == ""
+            ${build_name} =    Start New Build    namespace=redhat-ods-applications
+            ...    buildconfig=${build_configs}[${ind}]
+        ELSE
+            ${build_status} =    Get Build Status    namespace=redhat-ods-applications
+            ...    build_search_term=${build_name}
+            IF    "${build_status}" == "Failed"
```

Comment:
build_status  can be also == "Error" when builds don't work well

Reply:
Updated